README.md

```diff
@@ -30,3 +30,91 @@ TODO :
   present is an update from the v130 master branch only).
 - Do something about the 3rd party initiatives to provide some Docker or other 
   Plug and Play versions.
+
+## Installation
+
+[Download](https://github.com/oscarpilote/Ortho4XP/archive/refs/heads/master.zip) this repository and extract it to a preferred location on your computer and follow the instructions for your operation system below.
+
+### Windows
+
+1. Download and install [Python 3.11.9](https://www.python.org/downloads/release/python-3119/). Use the "Customize installation" option and make sure "pip" and "tcl/tk and IDLE" are selected on the Optional Features page.
+
+2. From a Command Prompt window, install the required packages:
+```py -3.11 -m pip install -r requirements.txt```
+
+3. Go to the Ortho4XP folder:
+```cd /path/to/Ortho4XP```
+
+4. Launch Ortho4XP:
+```py -3.11 -m Ortho4XP_v130.py```
```

Comment:
`Ortho4XP_v130.py` doesn't exist.

Try this instead. Since `Ortho4XP.py` is a script and not a module, the `-m` switch can be removed.
`py -3.11 Ortho4XP.py`